WordPress/Classes/ViewRelated/Media/GravatarOverlayView.swift

```diff
@@ -27,23 +27,23 @@ class GravatarOverlayView : UIView
         let ellipseRect = bounds.insetBy(dx: delta, dy: delta)
 
         // Setup
-        CGContextSaveGState(context)
-        CGContextSetLineWidth(context, borderWidth)
-        CGContextSetAllowsAntialiasing(context, true)
-        CGContextSetShouldAntialias(context, true)
+        CGContextSaveGState(context!)
```

Comment:
Can we just have `let context = UIGraphicsGetCurrentContext()!` above instead of adding `!`s all over the place?